.codesandbox/ci.json

```diff
@@ -2,5 +2,5 @@
   "buildCommand": "build:codesandbox",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 5 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26748&commitId=d30fb39e1cf3b0d97805b5c86c8552d0d1e90354&env=prod)

✅ There was 1 screenshots added, 4 screenshots removed, 1040 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- center, image larger.default.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>